Apex and SOQL doc Update (#4558)
forcedotcom/salesforcedx-vscode: docs/_articles/ja/apex/apex-tasks.md

---
title: Apex Tasks
lang: en
---

> This is a repository of sample tasks using Apex that you can complete in Code Builder or VS Code Extensions for desktop. We’ll keep adding to this list, so make sure you visit this topic often. 

Reviewer: ```suggestion
This is a repository of sample tasks using Apex that you can complete in Code Builder or VS Code using the Salesforce Extensions. We’ll keep adding to this list, so make sure you visit this topic often. 
```
